Makefile

```diff
@@ -649,6 +650,7 @@ pr\#%: clean-all
 	$(GO) run contrib/pr/checkout.go $*
 
 .PHONY: golangci-lint
+unexport CGO_CFLAGS
 golangci-lint:
```

Comment:
We normally don't put stuff between `.PHONY` and the target so this should move or be removed.

Can't we handle `CGO_CFLAGS` exactly like `LDFLAGS` with the export/unexport? Why even unexport?